### PR TITLE
Redesigned login and staff login page and made the ui changes in mess

### PR DESCRIPTION
**Pull request by mtabishk**
This PR introduces the following changes:
1. Redesigned login and staff login page
2. Added dynamic mess names instead of hardcoded values
3. Added name in mess manager complaint and feedback side instead of the email address
4. Added dropdown for mess names in the tender dialog box

https://user-images.githubusercontent.com/61789893/195812246-6aa03fee-8c67-496d-9e7b-ff98eb35f630.mp4

https://user-images.githubusercontent.com/61789893/195812206-98c48e3f-4455-4432-a15f-3e779ba97f05.mp4

### CODE REVIEW

File: app/.env.symlink [hunk 1]
```diff
@@ -1 +0,0 @@
-../.env
```
Comment (ashutoshc8101) [inline app/.env.symlink:1]: don't remove this file

**Review by ashutoshc8101 — CHANGES_REQUESTED** [review]

File: app/.env.symlink (hunk 1 above)
Comment (mtabishk) [reply app/.env.symlink:1]: @ashutoshc8101 Done

Comment (ashutoshc8101) [reply app/.env.symlink:1]: The change shouldn't be reflected here. We have handled this multiple times now.

**Review by ashutoshc8101 — CHANGES_REQUESTED** [review]

File: app/pubspec.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ dependencies:
   flutter_dotenv: ^5.0.2
   flutter_flavor: ^3.0.3
   localstorage: ^4.0.0+1
-  # qr_code_scanner: ^0.7.0
+  qr_code_scanner: ^0.7.0
```
Comment (ashutoshc8101) [inline app/pubspec.yaml:45]: Revert this

File: app/assets/google_fonts/OFL.txt [hunk 3]
```diff
@@ -1,93 +1,93 @@
-Copyright (c) 2010-2014 by tyPoland Lukasz Dziedzic (team@latofonts.com) with Reserved Font Name "Lato"
-
```
Comment (ashutoshc8101) [inline app/assets/google_fonts/OFL.txt:2]: Revert changes to this file.

File: app/.env.symlink [hunk 4]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+.env
```
Comment (ashutoshc8101) [inline app/.env.symlink]: This is still not fixed. Make sure to do a self review before asking for review.

**Review by ashutoshc8101 — CHANGES_REQUESTED** [review]

**Comment by ashutoshc8101** [comment]
Flutter analyze is failing.

File: app/pubspec.yaml (hunk 2 above)
Comment (mtabishk) [reply app/pubspec.yaml:45]: reverted

**Comment by mtabishk** [comment]
> Flutter analyze is failing.

@ashutoshc8101 Flutter analyze is failing due to commenting out  qr_code_scanner package. It is required by some pages

**Review by ashutoshc8101 — APPROVED** [review]

**Pull request merged**